src/Selector/Filepath.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace PHPat\Selector;
+
+use PHPStan\Reflection\ClassReflection;
+
+final class Filepath implements SelectorInterface
+{
+    private string $filename;
+    private bool $isRegex;
+
+    public function __construct(string $filename, bool $isRegex)
```

Comment:
I think `$filename` should be changed to `$filepath` since we are expecting the full path, not only the name of the file

Reply:
Yep, you're correct! Sorry, I originally had it all named Filename, and then came to this same conclusion and missed a few references to the old name :shakefist: